internal/namespaces/autocomplete/autocomplete.go

```diff
@@ -64,6 +64,7 @@ var autocompleteScripts = map[string]autocompleteScript{
 		CompleteScript: `eval "$(scw autocomplete script shell=bash)"`,
 		ShellConfigurationFile: map[string]string{
 			"darwin": path.Join(os.Getenv("HOME"), ".bash_profile"),
```

Comment:
maybe we should use `.bashrc` for macOS too ? 

I have both `.bashrc` and `.bash_profile`

Reading this, https://apple.stackexchange.com/a/51038, I'm not sure which is best.

Reply:
`.bash_profile` is widely used in macOS, we should use it.